unit-tests/non_func/test-leak-detection.cpp

```diff
@@ -0,0 +1,395 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2015 Intel Corporation. All Rights Reserved.
+
+//#cmake: static!
+
```

Comment:
Add at least one `//#test:device D400*` line for this test to be considered "live"